### DISCUSSION TITLE
GPU or CPU

### DISCUSSION THREAD

**Discussion by ThorstenUm** [blakeblackshear/frigate#12573] [General]
Hello,
I am running 4 Reolink RLC-811A cameras on an MSI Cubi N 8GL-001BEU Mini PC Barebone Intel Celeron N4000 on Frigate 0.14.0 Beta 4. A Coral is also installed.

Under "General" the GPU insert "intel-qsv" is always running between 1 and 1.5%. The total CPU load is shown as 40 to 50%. The Coral has a load between 5 and 30%, depending on the event situation.

It is strange that under the "Cameras" tab the 4 cameras are shown with CPU and not with GPU. One camera, for example, runs with ffmpeg 5%, capture with 1.5% and detect with 3.7%. The other cameras run similarly. "ffmpeg" is entered in the configuration with "hwaccel_args: preset-intel-qsv-h264". Shouldn't ffmpeg run on GPU instead of CPU, as shown under "Cameras"?

**Comment by NickM-27** [comment]
> The Coral has a load between 5 and 30%, depending on the event situation.

to be clear this is not the load of the coral (as that does not exist), it is the CPU used to process the images and send them to the coral for inference

> It is strange that under the "Cameras" tab the 4 cameras are shown with CPU and not with GPU.

It is not, the GPU only handles decoding and potentially scaling of the video feed. The CPU still has to be used to move the video data between the GPU and system RAM, it also has to manage motion detection, processing each frame (cropping, scaling), etc.

**Comment by ThorstenUm** [comment]
So the values ​​I described for GPU, CPU and Coral are normal for the hardware? I just wanted to make sure that I haven't made a configuration error...

**Reply by NickM-27** [reply]
yes, based on the info provided sounds like it is running fine

**Comment by ThorstenUm** [comment]
Thank you very much and keep it up. Frigate really is a great system!
